recipes/wtdbg/meta.yaml

```diff
@@ -15,6 +15,8 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+  host:
+    - zlib
```

Comment:
You missed 
```
run:
- zlib 
```

Reply:
Ah no, you don't need it. Just replace 2 lines in build.sh as the tool does not use ./configure